data/transactions/logic/opcodes.go

```diff
@@ -85,15 +86,41 @@ const spOpcodesVersion = 11 // falcon_verify, sumhash512
 // Unlimited Global Storage opcodes
 const boxVersion = 8 // box_*
 
+// CustomCost encapsulates a custom cost function and its documentation, applicable to opcodes like bmodexp where linearCost is inadequate by itself
+type CustomCost struct {
+	compute func(stack []stackValue, depth int) int
+	docCost string
+}
+
+// Custom cost definition for bmodexp
+func bmodExpCostFunction(stack []stackValue, depth int) int {
+	last := len(stack) - depth - 1 // mod
+	prev := last - depth - 1       // exp
+	pprev := last - depth - 2      // base
+
+	expLength := float64(len(stack[prev].Bytes))
+	modLength := float64(len(stack[last].Bytes))
+	baseLength := float64(len(stack[pprev].Bytes))
+	cost := (math.Pow(math.Max(baseLength, modLength), 1.63) * expLength / 15) + 200
```

Comment:
Wouldn't be better to declare those parameters (`1.63`, `15` and `200`) as constants? I think it would make easier to "tune" them in one place, if necessary, instead of replacing them in all the consumers.

Maybe also some comments on how those constants have been chosen would be helpful for future readers.

Reply:
Thanks for addressing this. Looks good!